package.json

```diff
@@ -1979,6 +1979,12 @@
           "default": 30000,
           "minimum": 1
         },
+        "testMate.cpp.test.showStderrArrows": {
```

Comment:
`"testMate.cpp.test.showStderrArrows"` -> `"testMate.cpp.test.stderrDecorator"`

Because it is a bit more general and in case the arrow changes this name would not fit anymore.